frame/dapps-staking/src/lib.rs

```diff
@@ -78,29 +41,6 @@ pub struct ActiveEraInfo {
     start: Option<u64>,
 }
```

Comment:
ActiveEraIndex is not used. There is code in on_finalize but that is not used as well

Reply:
Will remove.